### ISSUE TITLE
Supply vendor for BG Rep

### ISSUE THREAD

**Issue by Kelzior** [classicdb/database#819]
![image](https://cloud.githubusercontent.com/assets/9686159/13207995/d16c4700-d914-11e5-8fbb-718ab0cb8765.png)

Horde defilers bg supply vendor requires "alliance" rep, which is wrong
same for warsong supply, league of arathor supply vendors ..

**Comment by cala** [comment]
Indeed, some items were inverted. Thank you for reporting. I added this to our BG milestone: https://github.com/classicdb/database/milestones/Stand%20Your%20Ground

**Comment by cala** [comment]
Partly fixed in https://github.com/classicdb/database/commit/ac49ff097cb2661e121f6847a3c633ef77f5115f
However, the armor/weapon items cannot be fixed in the current state of the DB/Core because those items have two faction requirements depending on the player's team (on faction for Horde, one for Alliance). Currently, the core/DB can only handle one single faction. This is a specific case for Classic PvP pre-2.0.1
We need to update the core first to be able to fix this, but I don't know what would be the best way to do it.

**Comment by cala** [comment]
Warsong reputation items are fixed in https://github.com/cmangos/classic-db/commit/f069e5cc8671dcd51c6b496627159e1af869abe9
